Makefile

```diff
@@ -83,22 +82,23 @@ $(WWW_DIR):
 # Tests
 #
 
-test: unittest testmanual
+test: test-style unittest testmanual
 
-
-ut-%: build
```

Comment:
I removed this as I did not see it being used anywhere. I can revert this if a maintainer feels it should be brought back